charts/oes/templates/opa-deploy.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
```

Comment:
we should made this conditional...on by default and the user can turn if off if needed.

Reply:
Done.